forge-gui/res/cardsfolder/upcoming/sting_the_glinting_dagger.txt

```diff
@@ -0,0 +1,9 @@
+Name:Sting, the Glinting Dagger
+ManaCost:2
+Types:Legendary Artifact Equipment
+K:Equip:2
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ 3 | AddToughness$ 1 | AddKeyword$ Haste | Description$ Equipped creature gets +1/+1 and has haste.
+T:Mode$ Phase | Phase$ BeginCombat | TriggerZones$ Battlefield | Execute$ TrigUntap | TriggerDescription$ At the beginning of each combat, untap equipped creature.
+SVar:TrigUntap:DB$ Untap | Defined$ Equipped 
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddKeyword$ First Strike | IsPresent$ Creature.blockedBySource+Goblin,Creature.blockedBySource+Orc,Creature.blockingSource+Goblin,Creature.blockingSource+Orc | Description$ Equipped creature has first strike as long as it’s blocking or blocked by a Goblin or Orc.
```

Comment:
Source doesn't make sense because it refers to the equipment
Try:
`Creature.EquippedBy+blockingValid Goblin,Creature.EquippedBy+blockingValid Orc,Goblin.blockingValid Creature.EquippedBy,Orc.blockingValid Creature.EquippedBy`

Reply:
Thanks seems to work ! 
![image](https://github.com/Card-Forge/forge/assets/67333662/ef442f5c-7bae-4b9a-8520-f3a1e8f57ebd)